data/repository/src/main/java/com/chocolate/repository/datastore/remote/DirectMessageRemoteDataSource.kt

```diff
@@ -0,0 +1,15 @@
+package com.chocolate.repository.datastore.remote
+
+import com.chocolate.entities.directMessage.DMMessage
+import com.chocolate.repository.model.dto.direct_message.Chat
+import kotlinx.coroutines.flow.Flow
+
+interface DirectMessageRemoteDataSource {
+
+    suspend fun getChatsByUserId(userid: String, currentOrgName: String): Flow<List<Chat>>
```

Comment:
change the parameter `currentOrgName` to `currentOrganizationName`
use member instead of user